typescript/clockify/data.json

```diff
@@ -0,0 +1,9 @@
+{
+    "apikey":"YOUR-API-KEY-HERE",
```

Comment:
can we pass `apiKey` as an argument to the Sequence? It would be along the security guidance we recommend.